packages/terra-consumer-nav/src/components/popup/Popup.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import IconClose from 'terra-icon/lib/icon/IconClose';
+import TerraPopup from 'terra-popup';
+import styles from './Popup.scss';
+
+const cx = classNames.bind(styles);
+
+
+const propTypes = {
+  /**
+   * Title to be rendered on top of the popup content
+   */
+  title: PropTypes.oneOfType([
```

Comment:
In general, the terra team tries to stray away from prop types that can be multiple very different types